test/cram/interactive_viewer_dummy.ml

```diff
@@ -0,0 +1,56 @@
+open Cmdliner
+
+let input_events : Notty.Unescape.event list ref = ref []
+let output_image : Notty.image ref = ref Notty.I.empty
+
+let open_dummy_terminal () : unit =
+  let input_fd = Unix.openfile "/dev/null" Unix.[O_RDWR] 0o666 in
+  let output_fd = Unix.dup Unix.stdout in
+  Unix.dup2 input_fd Unix.stdin;
+  Unix.dup2 output_fd Unix.stdout;
+  Unix.close input_fd;
+  Unix.close output_fd
+
+
+let output_image : Notty.image = !output_image
+
+let print_image () : unit =
+  Notty_unix.output_image output_image;
+  print_endline "Hello world";
```

Comment:
Could you try putting this into the image with [`Notty.I.string`](https://ocaml.org/p/notty/latest/doc/Notty/I/index.html#val-string) ?